src/core/CoreDatabaseManager.jl

```diff
@@ -5,18 +5,44 @@ mutable struct CoreDatabaseManager <: AbstractCoreDatabaseManager
 end
 
 function get_database(client::T, name::String)::CoreDatabase where {T<:AbstractCoreClient}
+    isempty(name) && throw(GraknClientException(CLIENT_MISSING_DB_NAME))
     if contains_database(client, name)
         return CoreDatabase(name)
     else
         throw(GraknClientException(CLIENT_DB_DOES_NOT_EXIST, name))
     end
 end
 
-function contains_database(client::T, name::String)::Bool where {T<:AbstractCoreClient}
+function contains_database(client::T, name::String) where {T<:AbstractCoreClient}
+    isempty(name) && throw(GraknClientException(CLIENT_MISSING_DB_NAME))
+    let db = DatabaseManagerRequestBuilder
+        req_result, status = databases_contains(client.core_stub.blockingStub, gRPCController() , db.contains_req(name))
+        return grpc_result_or_error(req_result, status, result->result.contains)
+    end
```

Comment:
No need to use a `let` block here. Let blocks are usually used to introduce a new local scope, and any variable introduced inside that block would not be visible to the code after the block.

Same comment for several other places below e.g. `get_all_databases`, `create_database`, and `delete_database`.

Reply:
Yeap. I will remove this where it is not necessary.